### ISSUE TITLE
Sentimental Classification for Multiple Sentences

### ISSUE THREAD

**Issue by KaiyeYang-git** [flairNLP/flair#2872]
I have understood how to classify one sentence's sentiments from '''TextClassifier.load('sentiment-fast')'''
However, how can I do multiple sentences for their separate sentiments for having a fast processing speed?
I know a for loop is functional for this case it is sooo slow

the example I am doing now is still based on one sentence:
scaler = MinMaxScaler(feature_range=(-1, 1))
flair_score=[]
def flairSentimentAnalysis(sentiment_text):
    classifier = TextClassifier.load('sentiment-fast')
    sentence = Sentence(sentiment_text)
    classifier.predict(sentence)
    sentence.labels[0].to_dict()
    attitude=sentence.labels[0].value
    score=sentence.labels[0].score
    if attitude=='NEGATIVE':
        score=-score
    return score

for text in raw_data['new_processed_text'][0:10]:
    score=flairSentimentAnalysis(text)
    flair_score.append(score)
flair_attitude_score=np.array(flair_score).reshape(-1,1)
flair_result=scaler.fit_transform(flair_attitude_score).flatten()

**Comment by KaiyeYang-git** [comment]
I have put all my sentences in a list and it can only generate one result which represents the sentiments of all sentence, but not the sentiments of each sentence.

![Screenshot 2022-07-25 143850](https://user-images.githubusercontent.com/92261453/180790818-f49b9867-a15d-4d87-9683-902de562bc05.png)
